content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+
+F﻿rom an infrastructure standpoint, a minimum of environments are necessary. You will need a test/ dev,  a staging and at least one production environment. The HPE Developer Community actually started with only a test/dev/staging on one side and a production on the other side.
+
+I﻿ will not focus here on the first steps. I leave it to you to figure out new subjects. I will talk a bit again about the infrastructure and especially the dedicated scripts and variables that you need to create to support the lifecycle of the workshop. As usual, there are two sides to the workshop's creation. What should be done on the backend and what needs to be done on the frontend (web portal and database server mainly)
+
+![](/img/wod-blogserie3-archi3.png "WOD Overview.")
+
+I﻿ will consider two scenarios going further. In the first one, I will create a simple workshop that does not require any infrastructure but the JupyterHub itself. Then, in a second phase, I will go through a more complex workshop creation process that will cover most of the possible cases I could think of.
+
+# S﻿imple workshop example:
+
+l﻿et's consider that I plan to create a new workshop on the Go language. It becomes more and more popular among the developer community I interact with and one of the the developer was kind enough to agree with working with me on creating this new workshop. After a first meeting, where I explainied to him the creation process, and the expectations, we quickly started to work together. We defined title, abstract,  notebooks' folder name, and student range. As far as infrastructure's requirements, a new kernel was needed. No additional scripts are requiered for this workshop.
+
+A﻿s an admin of the Workshops-on-demand infrastructure, I had to perform several tasks:
+
+##### O﻿n the backend server:
+
+1. ###### Test and validate installation of the new kernel on the staging backend server by:
+
+* * Creating a new branch for this test
+
+  * M﻿odifying the [backend server installation yaml file ](https://github.com/Workshops-on-Demand/wod-backend/blob/main/ansible/install_backend.yml#L326)to include the new kernel.
+
+  * Validating the changes by testing a new backend install process.
+
+  * Pushing the changes to the github repo.
+
+2. ###### Creating a user for the workshop developer on the test/dev and staging backend servers.
+3. ###### Providing to the developer the necessary information to connect to the test/dev and staging backend servers.
+4. ###### Copy over the workshop developer's home folder a workshop template containing examples of introduction, conclusion and lab notebooks, allowing him to start his work.
+5. **Give the developer the wod-notebook repo url for him to fork the repo and work locally on his machine (when workshop does not require appliance but just a jupyter kernel for instance)**
```

Comment:
Add the word "the" prior to "workshop" in the parentheses".
Add the word "an" prior to "appliance"
Change "jupyter" to "Jupyter"